File: src/test/java/eotree/TestEOComment.java
package eotree;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

/**
 * EO comments tests.
 */
public class TestEOComment {

    @Test
    public void testGenerateEOZeroIndent() {
        var comment = new EOComment("test comment");
        assertEquals(comment.generateEO(0), "# test comment");
    }

    @Test
    public void testGenerateEONonZeroIndent() {
        var comment = new EOComment("test comment");
        assertEquals(comment.generateEO(2), "# test comment");
    }
}

File: src/test/kotlin/eotree/TestEOComment.kt
package eotree

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.polystat.j2eo.eotree.EOComment

/**
 * EO comments tests.
 */
class TestEOComment {
    @Test
    fun testGenerateEOZeroIndent() {
        val comment = EOComment("test comment")
        Assertions.assertEquals(comment.generateEO(0), "# test comment")
    }

    @Test
    fun testGenerateEONonZeroIndent() {
        val comment = EOComment("test comment")
        Assertions.assertEquals(comment.generateEO(2), "# test comment")
    }
}
